src/containers/App/App.container.tsx

```diff
@@ -0,0 +1,29 @@
+import { connect } from 'react-redux'
+
+import App from './App'
+import { MapStateProps, AppProps } from './App.types'
+import { isRoot } from '../../modules/location/selectors'
+
+const mapState = (state: any): MapStateProps => {
+  return {
+    isHomePage: isRoot(state)
+  }
+}
+
+const mapDispatch = () => ({})
+
+const mergeProps = (
+  stateProps: MapStateProps,
+  dispatchProps: object,
+  ownProps: AppProps
+) => ({
+  ...stateProps,
+  ...dispatchProps,
+  ...ownProps
+})
```

Comment:
Question: I don't get why is this here, isn't this equal to the default mergeProps?

Reply:
nope, the default `mergeProps` is `Object.assign({}, ownProps, stateProps, dispatchProps)`, so you can't override the connected props